scripts/ssm-build-deb.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Execute the following as root to install lintian and fpm
+# apt-get install lintian
+# apt-get install ruby ruby-dev rubygems build-essential
+# gem install --no-ri --no-rdoc fpm
+
+# Then run this file, altering the version.
+# This file will create two versions of the deb file:
+# - apel-ssm_<version>_all.deb contains all the files necessary to run a
+#   the SSM as a sender.
+# - apel-ssm-service_<version>_amd64.deb will install service files
```

Comment:
Why is this one AMD 64 specific?

Reply:
Because this is the more system specific / service files. The exact value should probably be a configurable option.

Reply:
Oh, you mean just because this is what we use?

Reply:
Yes

Reply:
although it looks like we can package it as `all`, the deb equivalent of `noarch`

Reply:
Ah, ok, that would seem to be more consistent with the RPMs.

Reply:
chaged